packages/ui/ThreeDots/index.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import { Icon } from "ui";
+import styled from "styled-components";
+
+type ThreeDotsProps = {
+  items: Array<string>;
+} & React.HTMLProps<HTMLSelectElement>;
```

Comment:
only `items` prop is used. I think that `React.HTMLProps<HTMLSelectElement>` can be removed

```suggestion
};
```